docs/query-help-style-guide.md

```diff
@@ -110,6 +110,18 @@ For example:
 
 If your query checks code for a CWE weakness, you should use the `@tags` element in the query file to reference the associated CWEs, as explained [here](query-metadata-style-guide.md). When you use these tags, a link to the appropriate entry from the [MITRE.org](https://cwe.mitre.org/scoring/index.html) site will automatically appear as a reference in the output HTML file.
 
+## Validating qhelp files
+
+Before making a pull request you should ensure the `.qhelp` file is well-formed and can be generated without problems. For example, this can be done locally with `codeql-cli` tool. In order to do so just start the `codeql-cli` with the appropriate parameters, as seen in the following example:
```

Comment:
```suggestion
Before making a pull request, please ensure the `.qhelp` files are well-formed and can be generated without errors. This can be done locally with the CodeQL CLI, as shown in the following example:
```